mods/shields-up/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Shields Up",
+  "author": "Direstorm, Friendly Neighbor, Takarada, Derpy Horse, That Someone, Pompano, Tufton",
```

Comment:
ES-Mod-Share is smart now! You can add each author as separate array elements, and it'll auto-format it.

This will let us, in the future, be able to sort mods by author, or filter, or whatever. 

`"author": ["Direstorm", "Friendly Neighbor", "Takarada", "Derpy Horse", "That Someone", "Pompano", "Tufton"],`